blocks/ageverification/ageverification.js

```diff
@@ -0,0 +1,81 @@
+import { decorateIcons } from '../../scripts/lib-franklin.js';
+import { setCookie } from '../../scripts/scripts.js';
+/* age verification overlay */
+
+export default async function decorate(block) {
+  block.textContent = '';
+
+  const resp = await fetch('/templates/ageverification.plain.html', window.location.pathname.endsWith('/templates/ageverification') ? { cache: 'reload' } : {});
+
+  if (resp.ok) {
+    const html = await resp.text();
```

Comment:
It seems like in the following you are reading out the information from the html and create new elements for all of it. Can't we work with the html we get from the template directly?

Reply:
i've tried to work with the html directly - i was not get it to work with css only. lots of manipulations in the dom are needed for making it work :-(.

Reply:
new document layout (verification.docx) used - please check